web/templates/unread_banner/mark_as_read_disabled_banner.hbs

```diff
@@ -1,15 +1,13 @@
-<div id="mark_as_read_turned_off_banner" class="home-error-bar info">
-    <p id="mark_as_read_turned_off_content">
+<div id="mark_as_read_turned_off_banner" class="main-view-banner home-error-bar info">
```

Comment:
This would be for a follow-up cleanup, but what does `home-error-bar` do here? I'm wondering if it's a stale CSS class.